NOnion/Network/TorGuard.fs

```diff
@@ -20,6 +20,27 @@ type internal GuardSendMessage =
         ReplyChannel: AsyncReplyChannel<OperationResult<unit>>
     }
 
+module ExceptionUtil =
+    let RunGuardJobWIthExceptionHandlling<'T> job : Async<'T> =
```

Comment:
@parhamsaremi typo here

Reply:
(2 typos actually) please read the name carefully

Reply:
fixed